src/main/scala/beam/router/RoutingModel.scala

```diff
@@ -92,6 +92,7 @@ object RoutingModel {
   case class BeamLeg(startTime: Long,
                      mode: BeamMode,
                      duration: Long,
+                     fare: Double = 0.0,
```

Comment:
Fare should be a part of an EmbodiedBeamLeg and _not_ a BeamLeg. I will explain my reasoning on this when I have a chance, but for now please move (or I think fare is already there).

Reply:
Integrated Leg cost with fare.